lib/merkleTreeHelper.ts

```diff
@@ -1,25 +1,32 @@
-import {BigNumber, utils} from 'ethers';
+import {utils} from 'ethers';
 const {solidityKeccak256} = utils;
 import crypto from 'crypto';
 
-interface land {
+export type SaleLandInfo = {
   x: number;
   y: number;
   size: number;
-  price: number;
+  price: string;
   reserved: string;
   salt?: string;
-  assetIds?: Array<number>;
-}
+  assetIds?: Array<string>;
+};
+
+export type SaltedSaleLandInfo = SaleLandInfo & {
+  salt: string;
+};
 
-interface asset {
+export type AssetGiveawayInfo = {
   reservedAddress: string;
-  assetIds: Array<BigNumber> | Array<string> | Array<number>;
+  assetIds: Array<string>;
```

Comment:
This might make the tests fail. not sure

Reply:
Ignore. I see you have updated the merkleTree test assetId types to string. I will need to update this in AssetGiveaway tests too.